Automattic/vip-cli
# Add a validation to check each line for ENGINE!=InnoDB

## Description

We want to ensure that our SQL validator ensures customers are using the InnoDB Engine on their tables.
This validation checks for ENGINE != InnoDB and warns the user of which lines in the SQL dump are impacting the result.

## Steps to Test

Example:

1. Check out this PR
1. `npm ci` if you haven't in a while
1. Test a file that passes: `npm run build && VIP_PROXY="" API_HOST=http://localhost:4000 node ./dist/bin/vip-import-validate-sql ./__fixtures__/client-file-uploader/db-dump-ipsum-67mb.sql` and note that no validations are triggered.
1. Test a file that fails: `npm run build && VIP_PROXY="" API_HOST=http://localhost:4000 node ./dist/bin/vip-import-validate-sql ./__fixtures__/client-file-uploader/db-dump-ipsum-67mb-myisam.sql` and note that the validation fails and shows two line numbers where ENGINE=MyISAM

---
_This PR was started by: [git wf pr](https://github.com/groupon/git-workflow/releases/tag/v1.3.2)_
